src/test/java/tests/SortQualifierTests.java

```diff
@@ -177,6 +242,48 @@ public void testSortQualifer5() {
                      issues.toString());
     }
 
+    /**
+     * Tests sort qualifier with milestone key. Milestones with due date are sorted from
+     * latest * to earliest, then comes milestones without due date and no milestone
+     */
+    @Test
+    public void testSortMilestone1() {
+        List<TurboIssue> issues = testModel.getIssues();
+        TurboIssue issue6 = new TurboIssue("testrepo/testrepo", 6, "Issue6");
+        TurboIssue issue7 = new TurboIssue("testrepo/testrepo", 7, "Issue7");
+        issue7.setMilestone(testModel.getMilestones().get(3));
+        issues.add(issue6);
+        issues.add(issue7);
+
+        Comparator<TurboIssue> comparator = getComparatorForSortQualifier("sort:milestone,id");
+
+        Collections.shuffle(issues);
+        Collections.sort(issues, comparator);
+
+        assertEquals("[#3 Issue3, #2 Issue2, #1 Issue1, #4 Issue4, #7 Issue7, #5 Issue5, #6 Issue6]",
+                      issues.toString());
```

Comment:
Could we compare ids here instead (`issues.map(getId)` against `Arrays.asList(1, 2, 3)`)? So we don't rely on TurboIssue::toString.